tests/test_api.py

```diff
@@ -41,14 +41,18 @@ def test_prefix_implicit_https(self):
         URL prefix given a host with no specified protocol.
         """
         with settings.runtime_values(host='33.33.33.33'):
-            self.assertEqual(client.prefix, 'https://33.33.33.33/api/v1/')
+            with mock.patch.object(Client, '_make_request') as req:
```

Comment:
Since `_make_request` is a requests private method, maybe we should wrap it with our own method, so there will be one single place in the code base to be modified in the future?